cub/cub/device/dispatch/tuning/tuning_scan.cuh

```diff
@@ -302,4 +336,7 @@ struct policy_hub
 template <typename AccumT, typename ScanOpT = ::cuda::std::plus<>>
 using DeviceScanPolicy = detail::scan::policy_hub<AccumT, ScanOpT>;
 
+using detail::scan::MakeScanPolicyWrapper;
+using detail::scan::ScanPolicyWrapper;
+
```

Comment:
Q: Why is public API exposure necessary for those types? We should limit exposing types if they are not intended to be used directly by our users.